### PR TITLE
Copy edits for enable a new user space

### PR DESCRIPTION
**Pull request by gkmahon**
Signed-off-by: Mahoney <genessa.k.mahoney@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/maintenance/enable-user-space.rst [hunk 1]
```diff
@@ -4,38 +4,35 @@ Create and enable a new user space
...
 
-* Create a new user
-* Update the OS to its most current version using `swupd`.
-* Install the most common applications for system administrators and
-  developers using bundles.
-* Setup a new user.
-* Add the new user to the `wheel` group.
-* Install a GUI using those `sudo` privileges.
+* Create a new user.
+* Install and update the OS software to its current version.
```
Comment (mltullis) [inline source/clear-linux/guides/maintenance/enable-user-space.rst]: Let's revert this line to the original. The swupd command is relevant here.
* Update the OS to its most current version using `swupd`.

File: source/clear-linux/guides/maintenance/enable-user-space.rst [hunk 2]
```diff
@@ -4,38 +4,35 @@ Create and enable a new user space
...
-to enter a new password and then retype the new password for the new user
-account you just created.
+Replace the <userid> with the name of the user account you want to create including the password for that user. The :command:`passwd` command prompts you
+to enter a new password. Retype the new password for the new user
+account just created.
 
-Install and update software
-===========================
+Install and update the OS software to its current version
+*********************************************************
```
Comment (mltullis) [inline source/clear-linux/guides/maintenance/enable-user-space.rst]: Change to "Install and update the OS"

File: source/clear-linux/guides/maintenance/enable-user-space.rst [hunk 3]
```diff
@@ -56,23 +53,23 @@ utility:
 
 .. note::
 
-   The image we installed may not be the latest version of |CL| available on
+   The image installed may not be the latest version of |CL| available on
```
Comment (mltullis) [inline source/clear-linux/guides/maintenance/enable-user-space.rst]: OK to keep original per pronoun usage guidelines at https://clearlinux.org/documentation/clear-linux/reference/collaboration/documentation/simple.

File: source/clear-linux/guides/maintenance/enable-user-space.rst [hunk 4]
```diff
@@ -56,23 +53,23 @@ utility:
...
    the server. However, whenever the command
    :command:`swupd bundle-add <bundle>` runs, the OS is updated to the latest
-   available version. Our website provides more `information about swupd`_.
+   available version. The website provides more `information about swupd`_.
```
Comment (mltullis) [inline source/clear-linux/guides/maintenance/enable-user-space.rst]: OK to keep original per pronoun usage guidelines at https://clearlinux.org/documentation/clear-linux/reference/collaboration/documentation/simple.

File: source/clear-linux/guides/maintenance/enable-user-space.rst [hunk 5]
```diff
@@ -4,38 +4,35 @@ Create and enable a new user space
...
-* Update the OS to its most current version using `swupd`.
-* Install the most common applications for system administrators and
-  developers using bundles.
-* Setup a new user.
-* Add the new user to the `wheel` group.
-* Install a GUI using those `sudo` privileges.
+* Create a new user.
+* Install and update the OS software to its current version.
+* Set up a new user and add the new user to the `wheel` group.
+
```
Comment (mltullis) [inline source/clear-linux/guides/maintenance/enable-user-space.rst]: This may be a diff artifact, but if this is a blank line, please delete.

File: source/clear-linux/guides/maintenance/enable-user-space.rst [hunk 6]
```diff
@@ -4,38 +4,35 @@ Create and enable a new user space
 ##################################
 
 This section provides steps to complete the following basic setup tasks for
-a newly installed |CL| system:
+a newly installed |CLOSIA| system:
 
-* Create a new user
-* Update the OS to its most current version using `swupd`.
-* Install the most common applications for system administrators and
```
Comment (mltullis) [inline source/clear-linux/guides/maintenance/enable-user-space.rst]: Let's keep this bullet (this line and the next).

File: source/clear-linux/guides/maintenance/enable-user-space.rst (hunk 2 above)
Comment (mltullis) [reply source/clear-linux/guides/maintenance/enable-user-space.rst]: Change "Install and update software" to "Install and update the OS"

**Comment by tpleavitt** [comment]
The note on line 57:
.. note::
 
    The image we installed may not be the latest version of |CL| available on
    the server. However, whenever the command
    :command:`swupd bundle-add <bundle>` runs, the OS is updated to the latest
    available version. Our website provides more `information about swupd`_.

I don't believe this is accurate anymore but I need to verify.  If it isn't then we need to remove this note as well.

**Comment by mltullis** [comment]
@gkmahon and @MCamp859 
For your remaining commit to this pull request, could you also remove the note @tpleavitt flags in his comment?

Text to be removed is the note on or near line 57, including the ..note directive:

.. note::

The image we installed may not be the latest version of |CL| available on
the server. However, whenever the command
:command:`swupd bundle-add <bundle>` runs, the OS is updated to the latest
available version. Our website provides more `information about swupd`_.

**Comment by mvincerx** [comment]
I agree with removing previous line 57. 

However, I'm not comfortable with the verbiage in present line 47. I suggest a revision: 
"Install the `os-clr-on-clr` bundle:"

Reason: As it's written, for a new user, to state:  "Install the `os-clr-on-clr` bundle using the software update utility:" may be confusing.  Whereas in reality it's true that every `bundle-add` is an update for CL, that specific command is not a "software **update** utility" as is `sudo swupd update`. 

See: https://clearlinux.org/documentation/clear-linux/guides/maintenance/update

In short, err on the side of simplicity.
